### DISCUSSION TITLE
创建模板后退出了程序，如何再次启动？

### DISCUSSION THREAD

**Discussion by k4m7v2pz** [koishijs/koishi#759] [Q&A] [answered]
admin@iZwaazn2pcc2olZ:~/koishi$ ls -la
total 20
drwxr-xr-x  4 admin admin 4096 Jul 19 16:34 .
drwxr-xr-x 10 admin admin 4096 Jul 20 09:14 ..
drwxr-xr-x  5 admin admin 4096 Jul 19 16:49 koishi-app
drwxr-xr-x  2 admin admin 4096 Jul 19 15:11 node_modules
-rw-r--r--  1 admin admin   86 Jul 19 15:11 yarn.lock

admin@iZwaazn2pcc2olZ:~/koishi/koishi-app$ ls -la
total 192
drwxr-xr-x   5 admin admin   4096 Jul 19 16:49 .
drwxr-xr-x   4 admin admin   4096 Jul 19 16:34 ..
-rw-r--r--   1 admin admin    147 Jul 16 00:41 .editorconfig
-rw-r--r--   1 admin admin     45 Jul 16 00:41 .env
drwxrwxr-x   3 admin admin   4096 Jul 19 15:15 .github
-rw-r--r--   1 admin admin    165 Jul 16 00:41 .gitignore
-rw-r--r--   1 admin admin    138 Jul 19 16:37 koishi.yml
drwxr-xr-x   2 admin admin   4096 Jul 20 09:13 logs
drwxr-xr-x 326 admin admin  12288 Jul 19 15:17 node_modules
-rw-r--r--   1 admin admin   1617 Jul 19 15:15 package.json
-rw-r--r--   1 admin admin   2097 Jul 16 00:41 README.md
-rw-r--r--   1 admin admin    239 Jul 16 00:41 tsconfig.base.json
-rw-r--r--   1 admin admin    845 Jul 16 00:41 tsconfig.json
-rw-r--r--   1 admin admin 134684 Jul 19 15:17 yarn.lock

**Comment by shigma** [comment, marked as answer]
在你的机器人目录重新输入 `npm run start` 或 `yarn start`。
